StoreForecasts/Program.cs

```diff
@@ -1,2 +1,93 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿
+using System;
+using System.Collections.Generic;
+using dotenv.net;
+using ApiSample.Schema;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace ApiSample {
+  record ForecastJson(
+    string OperatingArea,
+    DateOnly ForecastStartDate,
+    DateTime UtcRetrievalTimestamp,
+    DateTime UtcForecastTimestamp,
+    bool IsPinned,
+    int Idf,
+    List<ForecastValueJson> LoadForecast
+  ) {}
+
+  record ForecastValueJson(
+    DateOnly Date,
+    int DaysOut,
+    double Forecast
+  ) {} 
+
+  class Program {
+    static string ConnectionString => Environment.GetEnvironmentVariable("CONN_STRING")!;
+    static string ApiKey => System.Environment.GetEnvironmentVariable("MCAST_API_KEY")!;
+    static readonly string MCastDomain = "demo-gas.mea-analytics.tools";
```

Comment:
I'm wondering if we should put the URL as part of the environment variables to minimize the number of places where someone  would have to make manual edits

Reply:
Yeah I was wondering about that too.  As far as what's in the environment variables and what's in code, we can sort of split it based on what you have to change to run it on your system vs what can stay unchanged, or we can split it based on secrets vs non-sensitive data.  Right now it's the latter, but maybe the former is better.